Sources/SKTestSupport/SwiftPMTestProject.swift

```diff
@@ -108,7 +133,7 @@ package class SwiftPMTestProject: MultiFileTestProject {
         let dir = productsDirectory.appendingPathComponent("\(moduleName).build")
         let enumerator = FileManager.default.enumerator(at: dir, includingPropertiesForKeys: nil)
         while let file = enumerator?.nextObject() as? URL {
-          if file.pathExtension == "o" {
+          if linkFileList.contains(try file.filePath) {
```

Comment:
Should `linkFileList` be made into a Set? Although, out of curiosity, is there a reason not to just link everything in `linkFileList`?

Reply:
> Should `linkFileList` be made into a Set? 

Good idea. I’ll do it in a follow-up PR.

> Although, out of curiosity, is there a reason not to just link everything in `linkFileList`?

I had issues with that in the past: https://github.com/swiftlang/sourcekit-lsp/pull/1367#discussion_r1623045741